tests/installation/grub_utils.pm

```diff
@@ -0,0 +1,80 @@
+package grub_utils;
```

Comment:
Could you please move this change and related ones to the separate commit, so that changes to the schedules are isolated?

Reply:
Done.